tests/js/helpers/createStubs.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Creates stubs for:
+ *   - a project or projects
+ *   - organization owning above projects
+ *   - router
+ *   - context that contains org + projects + router
+ */
+export function createStubs(
+  {
+    organization: additionalOrg,
+    project: additionalProject,
+    projects: additionalProjects,
+    router: additionalRouter,
+  } = {}
+) {
+  const projects = (additionalProjects ||
+    (additionalProject && [additionalProject]) || [{}]
+  ).map(p => TestStubs.Project(p));
+  const [project] = projects;
+  const organization = TestStubs.Organization({
+    projects,
+    ...additionalOrg,
+  });
+  const router = TestStubs.router(additionalRouter);
+
+  const routerContext = TestStubs.routerContext([
+    {
+      organization,
+      project,
+      router,
+      location: router.location,
+    },
+  ]);
+
+  return {
+    org: organization,
+    organization,
```

Comment:
Why do you attach organization here twice - both as `org` and `organization`?

Reply:
This `createStubs` function feels quite specific to `organizationEvents` rather than a truly generic `createStubs` helper function - can we keep this scoped to that set of tests?

Reply:
Do many components have `org` and `organization` as well as `project` and `projects`? I assumed no, but could be wrong. `createStubs` is also a very generic name, a rename to something along the lines of `createOrganizationWithProjectStub` or something would help i think

Reply:
This is only used if called explicitly. It has `org` and `organization` because  we have tests that use one or the other. If a component does use an org, then I think this is a good shorthand (at the minimum) to get `org` + `routerContext`. It also bootstraps org to have a child project since that would better represent a real organization.

Reply:
Renamed to `initializeOrg`